packages/website/src/components/BigNumbers/BigNumbers.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { VStack } from '@coderscamp/ui/components/Stack';
+
+import { StatsWithTitle } from './StatsWithTitle';
+
+const amounts = [
+  ['7.', '1516', '200', '6'],
+  ['6526', '1022', '372', '502'],
+];
+const labels = [
+  ['edycja', 'zadań i materiałów', 'miejsc na kurs', 'projektów do stworzenia'],
+  ['zgłoszeń', 'osób na kursie', 'wydanych certyfikatów', 'zakodowanych projektów'],
+];
```

Comment:
I think it would be better to pass amount and labels as an objects. Later in code you will use something like 'amount.currentEdition' instead of 'amount[0]' what is more readable, same suggestion with nested arrays